packages/common/src/util.ts

```diff
@@ -68,3 +78,37 @@ export const isOneOf = curry(
 export const returnNullOrUndefinedOrDate = (
   dateVal: string | number | null | undefined
 ) => (isNil(dateVal) ? dateVal : new Date(dateVal));
+
```

Comment:
This is borrowed from https://github.com/siberiacancode/lodash-omitdeep/blob/main/src/omitDeepBy/omitDeepBy.ts

Reply:
Do you think it's worthwhile to add a comment that this is borrowed from that repo above?

Reply:
added comment